packages/core/src/utils/spanUtils.ts

```diff
@@ -180,7 +195,7 @@ function spanIsSentrySpan(span: Span): span is SentrySpan {
  * However, this has a slightly different semantic, as it also returns false if the span is finished.
  * So in the case where this distinction is important, use this method.
  */
-export function spanIsSampled(span: Span): boolean {
+export function spanIsSampled(span: { spanContext: Span['spanContext'] }): boolean {
```

Comment:
m: is this change necessary? I don't see this being called in added code (yet?). Generally I vote, we either we adapt this function or we directly compare against `TRACE_FLAG_SAMPLED` but don't do both (see comment in `spanUtils.ts`). Also narrowing the type here for some reason screamed breaking change initially to me but turns out it's not (tested [here](https://www.typescriptlang.org/play/?inlineSources=true&inlineSourceMap=true&ts=5.0.4#code/JYOwLgpgTgZghgYwgAgMoAc4mQbwFDLIDOmIAwgPbgQAeYAXMgBQCUyAvAHy5FwC26ADYQAJowBGFCsKwBfANwFkcESIAyoANaNWHbgDcKwEYsK8BwscknSIWRbLx4YAVxAIwwKsVIBJIqj8QqJMJFiMOGHkVJB0jBhYANoA5FGU1HTJALqybPiEhFAQYC5Q2ACE5VEAdGkxtGCstUGWeI54CFREYMTxpBy4SoQq6lo6bFy4sgA0Q8QtoozwgkQQswU+WOmxDMwT3Ew48xaLyJVEzSciuW1OnSBEttWCFADmoX4BCyKhLCyKQA)) 🤔

Reply:
True, that's not needed anymore because I'm comparing against `TRACE_FLAG_SAMPLED`. That's a leftover from before, where I was passing a `ReadonlySpan` or a `Span`, which both have a `spanContext`, even though the types do not completely match. However, should we still change the type here? As we really only need an object which has a `spanContext`.

Reply:
ok, then let's just leave it as-is. We can change it when necessary